client/components/Review.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const SingleReview = styled.div`
+  display: flex;
+  border: 1px solid black;
+  flex-direction: column;
+`;
```

Comment:
I see you went with styled components as well! How are you liking them? I was considering moving some of my styling blocks to a separate `.js` file and importing them into my components to make things more readable.

Reply:
@CSellers720 - Seemed more intuitive than CSS Modules to me, although I was reading that other people disagree vehemently haha. That definitely sounds like a good readability technique! I will keep in mind as well.